app/views/days/show.html.erb

```diff
@@ -5,6 +5,11 @@
     <% if @day <= Aoc.latest_day %>
       <span>·</span>
       <%= link_to "link to puzzle", Aoc.url(@day), target: :_blank, rel: :noopener, class: "link-explicit link-external" %>
```

Comment:
En voyant le screenshot, je pense qu'on peut enlever le "link to" dans le texte et conserver uniquement "puzzle"